### PR TITLE
Collective Cleanups February 2021

### PR DESCRIPTION
**Pull request by Lolligerhans**
Merge/pull with `--squash` to prepare single commit in staging index, commit with `git commit`.
Created empty following request from https://github.com/official-stockfish/Stockfish/pull/3301.

---

### What is a cleanup?

(Trivial) cleanups are changes which do not require testing, because their benefit is apparent to everyone. This can include
- Spelling and orthography corrections
- Code style harmonization/homogenization
- Commentary refinement
- (Other small changes of evident benefit)

### Participation

You have two options to add your own cleanups:
1. Create a separate PR on my branch [trivial-cleanups_Feb2021](https://github.com/Lolligerhans/Stockfish/tree/trivial-cleanups_Feb2021) to my fork of Stockfish. This allows me to to easily merge your ideas, if they require large or complex changes.
2. For simple changes, comment below and I add them for you.

### Legend

:eyes: = I have seen your comment
:+1: = Comment is resolved or discussed further below

### CODE REVIEW

**Comment by BM123499** [comment]
Missing `constexpr` on `score` in `movepick.cpp`

**Comment by BM123499** [comment]
@Lolligerhans, I meant inside... ;)

**Comment by Lolligerhans** [comment]
@BM123499 I saw you test now and figured! xD The function one is on the house. :+1:

File: src/evaluate.cpp [hunk 1]
```diff
@@ -423,7 +423,6 @@ namespace {
             score += BishopOnKingRing;
 
         int mob = popcount(b & mobilityArea[Us]);
-
         mobility[Us] += MobilityBonus[Pt - 2][mob];
 
         if (Pt == BISHOP || Pt == KNIGHT)
```
Comment (BM123499) [inline src/evaluate.cpp:428]: Can we use `constexpr` here as well?

**Review by BM123499 — COMMENTED** [review]
we are using `constexpr` in many places, but can they be used here?

**Comment by Lolligerhans** [comment]
> we are using `constexpr` in many places, but can they be used on line 428 of evaluate.cpp?

@BM123499 Please add a commit with a concrete suggestion. You will need to address the defined-not-used warning resulting from excluding all uses of some of the `constexpr` variables from the function body. I originally just omitted this case for simplicity (keeping all constexpr definitions in a block at start), but surely there are technical solutions for this problem.

**Comment by snicolet** [comment]
Line 271-272 of position.cpp should be formatted like this:

```
               && (   file_of(square<KING>(sideToMove)) == file_of(st->epSquare)
                   || !(blockers_for_king(sideToMove) & (st->epSquare + pawn_push(~sideToMove))));
```

**Comment by snicolet** [comment]
Lines 521-522 of position.cpp should be like this:

```
      return    !st->previous->blockersForKing[sideToMove] & from))
             && !aligned(from, to, square<KING>(us));
```

**Comment by snicolet** [comment]
Lines 660-661 of position.cpp should be formatted like this:

```
      return   st->previous->checkersBB 
            || (   rank_of(square<KING>(~sideToMove)) == rank_of(from)
                && (st->previous->blockersForKing[~sideToMove] & from));
```

**Comment by BM123499** [comment]
> Lines 521-522 of position.cpp should be like this:
> 
> ```
>       return    !(st->previous->blockersForKing[sideToMove] & from)
>              && !aligned(from, to, square<KING>(us));
> ```

@snicolet it changes the logic.

**Comment by Lolligerhans** [comment]
Adjusted whitespace to follow suggested indentation. Kept original expressions where suggested corrections differ (as pointed out by BM123499). I suppose these suggestions were accidental (?)

**Comment by snicolet** [comment]
Yes, sorry :-)
Corrected to just push the `||` operator on the second line

**Comment by vondele** [comment]
fix copyright date in Makefile

**Comment by Lolligerhans** [comment]
> fix copyright date in Makefile

@vondele
(1) Adding for time until today or
```
 # Copyright (C) 2004-2008 Tord Romstad (Glaurung author)
 # Copyright (C) 2008-2015 Marco Costalba, Joona Kiiski, Tord Romstad
 # Copyright (C) 2015-2019 Marco Costalba, Joona Kiiski, Gary Linscott, Tord Romstad
+# Copyright (C) 2019-2021 The Stockfish developers (see AUTHORS file)
 #
 # Stockfish is free software: you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by
```
(2) replacing entire time span?
```
 # Stockfish, a UCI chess playing engine derived from Glaurung 2.1
-# Copyright (C) 2004-2008 Tord Romstad (Glaurung author)
-# Copyright (C) 2008-2015 Marco Costalba, Joona Kiiski, Tord Romstad
-# Copyright (C) 2015-2019 Marco Costalba, Joona Kiiski, Gary Linscott, Tord Romstad
+# Copyright (C) 2004-2021 The Stockfish developers (see AUTHORS file)
 #
 # Stockfish is free software: you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by
```

**Comment by snicolet** [comment]
I would go for option (2) replacing entire time span, as in other source files.

**Comment by snicolet** [comment]
for search.cpp:
• lines 1030-1034 (in the likelyFailLow block): trailing spaces at end of lines
• line 1083: trailing space
• line 1084: end of two lines comment is missing a point.

**Comment by snicolet** [comment]
Merged via 40cb0f076a62115af030c4524825d9ba73d61023, thanks :-)

**Pull request closed without merging**